src-cpp-structure/projects/App/Boot/DI.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <StatusServices/Accounts/Service.h>
+#include <StatusServices/WalletAccounts/Service.h>
+#include <StatusServices/Keychain/Service.h>
+
+#include <boost/di.hpp>
+
+namespace Status
+{
+    using namespace boost::di;
+
+    const auto Injector = make_injector(
```

Comment:
Each translation unit that includes this file will have its own instance/copy of the`Injector`. Please consider adding `inline` keyword.